default/python/AI/buildings.py

```diff
@@ -0,0 +1,48 @@
+import freeOrionAIInterface as fo
+from enum import Enum
+from typing import List
+
+from common.fo_typing import PlanetId
+from freeorion_tools.caching import cache_for_current_turn, cache_for_session
+
+
+class Building(Enum):
```

Comment:
Yeah, why not start to use some real objects.
BuildingType would be a better name for this one, I think.